From 75722f89290eb1cb316cd240763545d7277e0481 Mon Sep 17 00:00:00 2001
From: Werner Lemberg <wl@gnu.org>
Date: Wed, 11 Nov 2015 08:55:16 +0000
Subject: [cff, autofit] Switch off stem darkening by default.

* src/autofit/afmodule.c (af_autofitter_init), src/cff/cffobjs.c
(cff_driver_init): Do it.
---
diff --git a/src/autofit/afmodule.c b/src/autofit/afmodule.c
index 6bb8193..45fd360 100644
--- a/src/autofit/afmodule.c
+++ b/src/autofit/afmodule.c
@@ -371,7 +371,7 @@
 #ifdef AF_CONFIG_OPTION_USE_WARPER
     module->warping           = 0;
 #endif
-    module->no_stem_darkening = FALSE;
+    module->no_stem_darkening = TRUE;
 
     module->darken_params[0]  = CFF_CONFIG_OPTION_DARKENING_PARAMETER_X1;
     module->darken_params[1]  = CFF_CONFIG_OPTION_DARKENING_PARAMETER_Y1;
diff --git a/src/cff/cffobjs.c b/src/cff/cffobjs.c
index 0e0d5b0..d628b68 100644
--- a/src/cff/cffobjs.c
+++ b/src/cff/cffobjs.c
@@ -1052,7 +1052,7 @@
     driver->hinting_engine = FT_CFF_HINTING_ADOBE;
 #endif
 
-    driver->no_stem_darkening = FALSE;
+    driver->no_stem_darkening = TRUE;
 
     driver->darken_params[0] = CFF_CONFIG_OPTION_DARKENING_PARAMETER_X1;
     driver->darken_params[1] = CFF_CONFIG_OPTION_DARKENING_PARAMETER_Y1;
--
cgit v0.9.0.2

